commit dad9a599009617402c0771a539ac016f4dd1ef1a
Author: Paul Howarth <paul@city-fan.org>
Date:   Tue Jan 7 08:43:43 2025 +0000

    Remove broken test t/106_no_ignore_case.t
    
    Test t/106_no_ignore_case.t fails with Getopt-Long-Descriptive 0.116
    onwards. The equivalent test in MooseX-Getopt was subsequently removed
    in MooseX-Getopt 0.77 and replaced with a new test t/gld-getopt_conf.t.
    However, this new test checks the operation of the getopt_conf parameter
    of MooseX-Getopt, which is not supported by MouseX-Getopt. Hence there
    is no replacement test.

diff --git a/t/106_no_ignore_case.t b/t/106_no_ignore_case.t
deleted file mode 100644
index 72c3ff6..0000000
--- a/t/106_no_ignore_case.t
+++ /dev/null
@@ -1,57 +0,0 @@
-use strict;
-use warnings;
-use Test::More;
-use Mouse ();
-use Mouse::Meta::Class;
-
-foreach my $role (qw/
-    MouseX::Getopt
-    MouseX::Getopt::GLD
-    MouseX::Getopt::Basic
-/) {
-    Mouse::Util::load_class($role);
-
-    my $meta = Mouse::Meta::Class->create_anon_class(
-        superclasses => ['Mouse::Object'],
-    );
-    $meta->add_attribute('BigD', traits => ['Getopt'], isa => 'Bool',
-        cmd_aliases => ['D'], is => 'ro');
-    $meta->add_attribute('SmallD', traits => ['Getopt'], isa => 'Bool',
-        cmd_aliases => ['d'], is => 'ro');
-    $role->meta->apply($meta);
-
-    {
-        my $obj = $meta->name->new_with_options(
-                { argv => ["-d"], no_ignore_case => 1}
-            );
-
-        ok((! $obj->BigD), "BigD was not set for argv -d on $role");
-        ok($obj->SmallD, "SmallD was set for argv -d on $role");
-
-    }
-    ok($meta->name->new_with_options({ argv => ['-d'], no_ignore_case => 1})
-            ->SmallD,
-        "SmallD was set for argv -d on $role");
-    {
-        local @ARGV = ('-d');
-        ok($meta->name->new_with_options()->SmallD,
-            "SmallD was set for ARGV on $role");
-    }
-
-    ok($meta->name->new_with_options({ argv => ['-D'], no_ignore_case => 1})
-            ->BigD,
-        "BigD was set for argv -d on $role");
-
-    {
-        my $obj = $meta->name->new_with_options(
-                { argv => ['-D', "-d"], no_ignore_case => 1}
-            );
-
-        ok($obj->BigD, "BigD was set for argv -D -d on $role");
-        ok($obj->SmallD, "SmallD was set for argv -D -d on $role");
-
-    }
-}
-
-done_testing;
-
